fe/fe-core/src/main/java/org/apache/doris/persist/EditLog.java

```diff
@@ -1576,7 +1576,9 @@ public void logTableRename(TableInfo tableInfo) {
     }
 
     public void logModifyViewDef(AlterViewInfo alterViewInfo) {
-        logEdit(OperationType.OP_MODIFY_VIEW_DEF, alterViewInfo);
+        long logId = logEdit(OperationType.OP_MODIFY_VIEW_DEF, alterViewInfo);
+        LOG.info("log modify view, logId : {}, infos: {}", logId, alterViewInfo);
```

Comment:
Change this log level to debug, since the output of `AlterViewInfo` might be huge.